### DISCUSSION TITLE
【记录】终于在50级前把烈度11的妮妮薇打过了，发个帖子记录一下

### DISCUSSION THREAD

**Discussion by Wild-Alaska** [share121/inter-knot#2085] [其他]
![F6D457A1A7FAC55DD592F290EE3F534B](https://github.com/user-attachments/assets/d2181cac-a93f-47d5-9b2e-e35782424207)
血是真的厚啊，不过总的来说妮妮薇比第一二面的boss要简单的多，唯一比较难处理的就是p3开花的弹幕
贴一下面板
![45852951E6413674712BD7012AD45C2C](https://github.com/user-attachments/assets/4e9dec5d-aebc-4ae1-8ab5-bbde154f703e)
![141F3F774515F9D58F55E44799D5517D](https://github.com/user-attachments/assets/e9dc8c89-d14a-4c55-b1e7-46bca5f7845d)
![9A78F75549B5F69199FE2259DFBA5534](https://github.com/user-attachments/assets/bf80e9b0-9f79-403e-bf53-283fb0cea5ef)

**Comment by likeseele** [comment]
大佬，牛！
